tests/integration/sync/it_sync_customers_test.py

```diff
@@ -49,7 +48,68 @@ def test_sync_customers(customers_mock, auth_mock):
 
     sync_customers()
 
-    customer = Customer.query.get(poster_id=poster_customer["client_id"])
+    customer = Customer.query.filter_by(
+        first_name=poster_customer["firstname"],
+        last_name=poster_customer["lastname"]
+    ).one()
     assert customer.first_name == poster_customer["firstname"]
     assert customer.last_name == poster_customer["lastname"]
     assert customer.phone_number == poster_customer["phone_number"]
+
+
+@pytest.mark.skip(
+    "Skipped until puzzle below is not done")
+@unittest.mock.patch.object(Authenticated, 'auth')
+@unittest.mock.patch.object(Poster, 'customers')
+def test_sync_customers_with_merging_data(customers_mock, auth_mock):
+    """
+    @todo #409:30min Remove skipping decorator from this test.
+     Now def merge_data(model, poster_data, timelessis_data) func is not
+     working properly, pls fix it. After fixing this test should work
+    """
```

Comment:
@vladarefiev Why is that function not working?

Reply:
@amihaiemil I didn't investigate it. 
`sqlalchemy.orm.exc.FlushError: New instance <Customer at 0x7fd5b606e4a8> with identity key`